docs/index.rst

```diff
@@ -1,13 +1,12 @@
-Documentation
-=============
+setuptools
```

Comment:
How about 
```suggestion
|project|
```

Reply:
Have you tried running with that substitution? For me, it causes the build to fail with `/Users/jaraco/code/public/pypa/setuptools/docs/index.rst:4:Undefined substitution referenced: "project".`

Reply:
You need to set project in conf.py for that to work.

Reply:
Hmm. That value is [set in config by jaraco.packaging.sphinx](https://github.com/jaraco/jaraco.packaging/blob/38e707641eac9ccfe5f6b8679894dc68a000df20/jaraco/packaging/sphinx.py#L37-L40). My experience has been that setting the property there was equivalent to setting it in conf.py, but perhaps there's another thing that has to happen.

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#default-substitutions

Aha, project isn't a default substitution in Sphinx. :)

I shouldn't have assumed.

Reply:
Oh, I usually declare it so I forgot that it's not there by default